scipy/sparse/linalg/_eigen/lobpcg/lobpcg.py

```diff
@@ -155,71 +155,79 @@ def lobpcg(
 ):
     """Locally Optimal Block Preconditioned Conjugate Gradient Method (LOBPCG).
 
-    LOBPCG is a preconditioned eigensolver for large symmetric positive
-    definite (SPD) generalized eigenproblems.
+    LOBPCG is a preconditioned eigensolver for large real symmetric and complex
+    Hermitian definite generalized eigenproblems.
 
     Parameters
     ----------
-    A : {sparse matrix, dense matrix, LinearOperator, callable object}
-        The symmetric linear operator of the problem, usually a
+    A : {sparse matrix, ndarray, LinearOperator, callable object}
+        The Hermitian linear operator of the problem, usually given by a
         sparse matrix.  Often called the "stiffness matrix".
     X : ndarray, float32 or float64
-        Initial approximation to the ``k`` eigenvectors (non-sparse). If `A`
-        has ``shape=(n,n)`` then `X` should have shape ``shape=(n,k)``.
-    B : {dense matrix, sparse matrix, LinearOperator, callable object}
-        Optional.
-        The right hand side operator in a generalized eigenproblem.
-        By default, ``B = Identity``.  Often called the "mass matrix".
-    M : {dense matrix, sparse matrix, LinearOperator, callable object}
-        Optional.
-        Preconditioner to `A`; by default ``M = Identity``.
-        `M` should approximate the inverse of `A`.
-    Y : ndarray, float32 or float64, optional.
-        An n-by-sizeY matrix of constraints (non-sparse), sizeY < n.
-        The iterations will be performed in the B-orthogonal complement
-        of the column-space of Y. Y must be full rank.
-    tol : scalar, optional.
-        Solver tolerance (stopping criterion).
+        Initial approximation to the ``k`` eigenvectors (non-sparse).
+        If ``A`` has ``shape=(n,n)`` then ``X`` must have ``shape=(n,k)``.
```

Comment:
Parameters also need a single quote.
```suggestion
        Initial approximation to the ``k`` eigenvectors (non-sparse).
        If `A` has ``shape=(n,n)`` then `X` must have ``shape=(n,k)``.
```